examples/plugin/wasi-logging/README.md

```diff
@@ -0,0 +1,78 @@
+# WasmEdge WASI-Logging example.
```

Comment:
Please use a grammar checker such as Grammarly to correct some wording issues.

Reply:
Fixed